csi.proto

```diff
@@ -502,6 +510,34 @@ message ControllerServiceCapability {
     RPC rpc = 1;
   }
 }
+
+message ControllerExpandVolumeRequest {
+  // The ID of the volume to expand. This field is REQUIRED
+  string volume_id = 1;
+
+  // the current capacity of the volume in bytes.
+  // This field is OPTIONAL
+  int64 current_capacity_bytes = 2;
+
+  // This field is REQUIRED. This allows CO to specify the
+  // capacity requirements of the volume after expansion
+  CapacityRange capacity_range = 3;
+
+  // Secrets required by the plugin for expanding the volume.
+  // This field is OPTIONAL
+  map<string,string> controller_volume_expand_secrets = 4;
```

Comment:
Are there any storage systems that require per volume credentials for resizing? If not, I'd rather start without them and add them in the future, if needed.

Reply:
The credentials requirement of expand volume should fall inline with `CreateVolume` RPC call.  For example - unless CSI driver runs with a pre-configured username and password for connecting to heketi, a glusterfs volume can not be expanded without secrets.